radar-auth/src/test/java/org/radarcns/auth/util/TokenTestUtils.java

```diff
@@ -63,11 +63,9 @@ public static void setUp() throws Exception {
     }
 
     private static void initVars(String publicKey, Algorithm algorithm) {
-        PUBLIC_KEY_BODY = "{\n"
-            + "  \"alg\" : \"SHA256withRSA\",\n"
-            + "  \"value\" : \"-----BEGIN PUBLIC KEY-----\\n" + publicKey + "\\n-----END PUBLIC "
-            + "KEY-----\"\n"
-            + "}";
+        PUBLIC_KEY_BODY = "[\n"
+            + " \"-----BEGIN PUBLIC KEY-----\\n" + publicKey + "\\n-----END PUBLIC KEY-----\"\n"
+            + "]";
```

Comment:
Perhaps change this test as well? For the `alg` parameter, the spec uses values from [RFC 7518](https://www.rfc-editor.org/rfc/rfc7518.txt), which translates to `alg = RS256` and `kty = RSA` for RSA 256, and `alg = ES256` and `kty = EC` for ECDSA with `P-256` curve.

Reply:
Yes. Will do